src/app/[locale]/projects/page.tsx

```diff
@@ -25,35 +25,42 @@ interface ProjectType {
     state: string
 }
 
-let data: ProjectType[] = []
-
-const limit = 10
-const columns = ['Project Name', 'Description', 'Project Responsible', 'License Clearing', 'State', 'Actions']
-const noRecordsFound = 'No project data to show.'
-const advancedSearch = {
-    'Project Name': '',
-    'Project Version': '',
-    'Project Type': ['Customer Project', 'Internal Project', 'Product', 'Service', 'Inner Source'],
-    'Project Responsible (Email)': '',
-    Group: ['None'],
-    State: ['Active', 'PhaseOut', 'Unknown'],
-    'Clearing State': ['Open', 'In Progress', 'Closed'],
-    Tag: '',
-    'Additional Data': '',
-}
+function Project() {
+    const [data, setData] = useState([])
+    const t = useTranslations(COMMON_NAMESPACE)
 
-async function Project() {
-    const fetchData = (await sw360FetchData('/projects?allDetails=true', 'projects')) as ProjectType[]
-
-    if (fetchData !== null) {
-        data = fetchData.map((item) => ({
-            name: item.name,
-            description: item.description,
-            projectResponsible: item.projectResponsible,
-            state: item.state,
-        }))
+    const pagination = { limit: 10 }
+    const columns = [
+        t('Project Name'),
+        t('Description'),
+        t('Project Responsible'),
```

Comment:
Project Responsible is missing in language files

Reply:
Done